pkg/controller/dataimportcron-controller_test.go

```diff
@@ -369,6 +371,10 @@ func newImageStream(name string) *imagev1.ImageStream {
 		Status: imagev1.ImageStreamStatus{
 			Tags: []imagev1.NamedTagEventList{
 				{
+					Tag: "nosuch",
```

Comment:
Maybe also test the negative case where we have a tag that is not in the status.tags? Also are we still testing the no tag case?

Reply:
added